### ISSUE TITLE
Support Request: VAST 3 and controlBar

### ISSUE THREAD

**Issue by jgubman** [googleads/videojs-ima#227]
Hi, I'm having an issue with the videojs-ima controlBar and a preroll on our site. The advertiser requires their preroll to have it's own controlBar.

I set up a fiddle with the advertiser's adTagURL:
https://jsfiddle.net/3s1qjtub/1/
and noticed that there's something that sets the #ima-control-bar div to display: none;  If you inspect the video, that style appears to be appended to the div.

Unfortunately that behavior isn't being applied on my site. The controlBar continues to display and follow the regular mouseover/mouseout behavior. I also don't see anything in the code that would explain why the display:none is being added in the fiddle.

Do you know where that's coming from?

**Comment by shawnbuso** [comment]
Hey there,

Are you saying you're seeing the video.js plugin control bar over your VPAID ad? Because I'm actually not seeing that on my end. Even on mouse over, I'm not getting our control bar, just the ad's. Testing on Chrome on Linux, here's a screenshot of what I see while the ad is playing:

![screenshot](http://i.imgur.com/q55FPyc.png)

I do see the video.js control bar return when content starts, but it sounds like that shouldn't be an issue for you - correct me if I'm mistaken.

Thanks,
Shawn

**Comment by jgubman** [comment]
Yeah, on our site we see the videojs-ima plugin’s control bar over the VPAID ad, in the fiddle I don’t see the control bar, and I don’t see anything in the videojs-ima plugin’s source code that would hide the control bar (other than on mouseout).

I’d point you to and example on our site (http://www.fod-staging.com/videos/74/the-landlord-from-will-ferrell-and-adam-ghost-panther-mckay), but I can’t seem to get my ad trafficker to keep this stupid VPAID ad up and we only show prerolls every 3rd view so it can be frustrating to try and find the preroll there.

Would you expect the videojs-ima’s control bar to be hidden during a VPAID ad like the one on the fiddle?

**Comment by shawnbuso** [comment]
The fiddle behavior is the correct behavior - we hide the ad controls for anything trafficked as non-video [here](https://github.com/googleads/videojs-ima/blob/master/src/videojs.ima.js#L327). That was added back on 1/5 - is there a chance that you have a version of the plugin on your site from before then? I tried checking if that code is present on your site, but it looks like the video.js ima plugin is obfuscated.
